CHANGELOG.rst

```diff
@@ -41,6 +41,11 @@ Fixed
 * Fixed service discovery blocking forever if device disconnects in BlueZ backend. Merged #1092.
 * Fixed ``AttributeError`` crash when scanning on Windows builds < 19041. Fixes #1094.
 
+Fixed
+-----
+* BlueZ: Cancel the device discovery wait task if the device disconnects in
+  between to avoid a timeout
+
 Fixed
 -----
 * BlueZ: Cancel the device discovery wait task if the device disconnects in
```

Comment:
Something's wrong here, these added lines seem to exist already in the file?

Reply:
How did that happen :D

Reply:
Fixed it now. took the opportunity to cleanup the commits as well :)